web/styles/compose.css

```diff
@@ -545,16 +545,16 @@ input.recipient_box {
 }
 
 #compose-send-button {
-    padding: 3px 12px;
+    display: flex;
+    align-items: center;
+    padding: 0 12px;
     margin-bottom: 0;
     font-weight: 600;
     font-size: 0.9em;
     border-radius: 4px 0 0 4px;
 
     .loader {
         display: none;
-        position: relative;
```

Comment:
These are some really bold changes! It would be good as the PR matures to include screenshots demonstrating that these removals have no effect, or that their replacements are suitable and robust for things like loaders, spinners, etc.